src/MapzenSearchDataConverter.swift

```diff
@@ -0,0 +1,153 @@
+//
+//  MapzenSearchDataConverter.swift
+//  Pods
```

Comment:
Nit: Just noticed the `Pods` name here. This should be the project name, so `Mapzen-ios-sdk` or something similar.